command_line/combine_experiments.py

```diff
@@ -523,7 +530,13 @@ def average_detectors(target, panelgroups, depth):
                     params.output.min_reflections_per_experiment is not None
                     and n_sub_ref < params.output.min_reflections_per_experiment
                 ):
-                    skipped_expts += 1
+                    skipped_expts_min_refl += 1
+                    continue
```

Comment:
Code coverage report suggests that this is not tested currently - could be worth adding a test? There's probably plenty of useful example data in `dials_data` which could be used for this.